tests/test_consent/api/test_views.py

```diff
@@ -310,11 +311,21 @@ def _assert_expectations(self, response, expected_body, expected_status):
         ),
     )
     @ddt.unpack
-    def test_consent_api_get_endpoint(self, factory, items, request_body, expected_response_body, expected_status_code):
+    def test_consent_api_get_endpoint(
+            self,
+            factory,
+            items,
+            request_body,
+            expected_response_body,
+            expected_status_code,
+            catalog_api_client_mock,
```

Comment:
The naming of this feels a little weird. I think you can set the return value in the mock decorator to simplify things a bit and not have to do the renaming

Reply:
Good call. I meant to come back to this for cleanup after encountering an example of what you described 👍